### ISSUE TITLE
How to set time and date?

### ISSUE THREAD

**Issue by apanniello** [wizche/ttgo-smb#2]
Hi, just giving a try to your funny theme and was just trying to understand how I can setup time and date to the right timezone...
Is there something already implemented?
Otherwise, I could get inspired by some other themes that take the time from an ntp server and try to integrate it to your code?
Best Regards.

**Comment by wizche** [comment]
Ciao Attilio, actually there is nothing implemented yet, what happens at my side is that when I deploy the code to the device it seems to syncronize the RTC with my PC clock. Doesnt happen at ur side?
We definitely need a way to do it properly via NTP, bluetooth or manual setup (menu)

**Comment by apo76** [comment]
Ciao Sergio,

For some unknown reason, the RTC has not been updated with my PC clock at a first time....I had to try several times and then it worked....I've just sent a pull request where I have added NTP synchronisation, give it a look. 
Some remarks:
-when I move the clock, many time the screen stays off: any idea why? on you side also?
-could you we add a "double tap on screen" option that puts the screen on?

Best Regards,

Attilio

**Comment by wizche** [comment]
> -when I move the clock, many time the screen stays off: any idea why? on you side also?

I noticed that the wake up is not always working, sometimes you turn your wrist and the watch stay off. Seems like the gesture recognition on the BMA chip is not perfect, maybe there is some calibration one can do... (see [datasheet](https://github.com/Xinyuan-LilyGO/LilyGo-TWatch-DataSheet/raw/master/BMA423/BST-BMA423-DS000-1509600.pdf))

> -could you we add a "double tap on screen" option that puts the screen on?

Now, when u press the physical button the screen either turns on or off. What I noticed is that the BMA has an interrupt for tap/double tap events. Maybe we can wire those if you think they could help to wake-up the watch.
